factory-containers/assemble-system-image

```diff
@@ -150,6 +151,40 @@ def dump_apps_container_images(target: dict, containers_arch: str, app_image_tar
                                    '-C', os.path.join(tmp_docker_dir, containers_arch), '.'])
 
 
+def resize_wic(wic_image: str, increase_bytes: int):
+    increase_k = ceil(increase_bytes / 1024) + 1
+    wic_k = ceil(os.stat(wic_image).st_size / 1024)
+    count = wic_k + increase_k
+    cmd('dd', 'if=/dev/zero', 'bs=1024', 'of=' + wic_image,
+        'conv=notrunc', 'oflag=append', 'count=' + str(count),
+        'seek=' + str(wic_k))
```

Comment:
if seek == wic_k then count should be increase_k?

Reply:
that was how @mike-scott 's script worked. I think `count` and `seek` aren't related. ie - its going add count-seek bytes. right  mike?

Reply:
-rw-r--r--  1 mike mike 911M Aug 28 21:04 17b406dc02163a515da0a05f8388c4ffea492cda-arm.tar
-rw-rw-r--  1 mike mike 699M Aug 28 10:56 lmp-factory-image-apalis-imx6.wic.org
-rw-rw-r--  1 mike mike 2.3G Aug 28 10:39 lmp-factory-image-apalis-imx6.wic

2.3 = 699 + (699 + 911)